frontend/pages/ControlsPage/ControlsWrapper.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useContext } from "react";
+
+import { Tab, Tabs, TabList } from "react-tabs";
+import { InjectedRouter } from "react-router";
+import PATHS from "router/paths";
+import { AppContext } from "context/app";
+
+import mdmAppleAPI from "services/entities/mdm_apple";
+
+import TabsWrapper from "components/TabsWrapper";
+import MainContent from "components/MainContent";
+import TeamsDropdownHeader, {
+  ITeamsDropdownState,
+} from "components/PageHeader/TeamsDropdownHeader";
+import EmptyTable from "components/EmptyTable";
+import Button from "components/buttons/Button";
+
+import { IMdmApple } from "interfaces/mdm";
+
+import { find } from "lodash";
+import { useQuery } from "react-query";
+import Spinner from "components/Spinner";
+
+interface IControlsSubNavItem {
+  name: string;
+  pathname: string;
+}
+
+const controlsSubNav: IControlsSubNavItem[] = [
+  {
+    name: "macOS updates",
+    pathname: PATHS.CONTROLS_MAC_UPDATES,
+  },
+  {
+    name: "macOS settings",
+    pathname: PATHS.CONTROLS_MAC_SETTINGS,
+  },
+];
+
+interface IControlsWrapperProp {
+  children: JSX.Element;
+  location: any; // no type in react-router v3
+  router: InjectedRouter; // v3
+}
+
+const getTabIndex = (path: string): number => {
+  return controlsSubNav.findIndex((navItem) => {
+    // tab stays highlighted for paths that start with same pathname
+    return path.startsWith(navItem.pathname);
+  });
+};
+
+const baseClass = "controls-wrapper";
+
+const ControlsWrapper = ({
+  children,
+  location,
+  router,
+}: IControlsWrapperProp): JSX.Element => {
+  const { availableTeams, isPremiumTier, setCurrentTeam } = useContext(
+    AppContext
+  );
+
+  const {
+    data: mdmApple,
+    isLoading: isLoadingMdmApple,
+    error: errorMdmApple,
+  } = useQuery<IMdmApple, Error, IMdmApple>(
```

Comment:
not sure but is there another source for telling if a user has apple MDM enabled? maybe a prop on the config? if not then this is fine.